<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en-us">

  <head>
  <meta charset="UTF-8">
  <link href="http://gmpg.org/xfn/11" rel="profile">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      European amphioxus genome &middot; F. Marlétaz
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-precomposed.png">
  <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/atom+xml" title="F. Marlétaz" href="/atom.xml">
</head>


  <body>

    <div class="container content">
      <div class="masthead">
        <h3 class="masthead-title">
          <a href="/" title="Home">F. Marlétaz</a>

          
              &nbsp;&nbsp;&nbsp;<small><a href="/about/">About</a></small>
          
              &nbsp;&nbsp;&nbsp;<small><a href="/Research/">Research</a></small>
          
              &nbsp;&nbsp;&nbsp;<small><a href="/Publications/">Publications</a></small>
          
              &nbsp;&nbsp;&nbsp;<small><a href="/Amphioxus/">Amphioxus</a></small>
          
              &nbsp;&nbsp;&nbsp;<small><a href="/atom.xml">Feed</a></small>
          

        </h3>
      </div>

      <article class="page">
  <h1 class="page-title">European amphioxus genome</h1>
  <div class="message">
  <b><u>Disclaimer:</u> This data is made available to the members of the european amphioxus genome consortium as collaborative material. These datasets are not to be used for whole genome analyses unless otherwise agreed for, and not to be redistributed to anybody outside the consortium without prior arrangments!!</b></div>

<h2 id="datasets">Datasets</h2>

<p>The genome of the european amphioxus <em>Branchiostoma lancealatum</em> was generated from 150x Illumina coverage sequenced by <a href="http://www.genoscope.cns.fr">Génoscope</a>. In the Bl71nemr assembly, haplotypes were reconcilied using haplomerger. </p>

<p>This assembly spans 495.3Mb (N50: 1.29Mb) split in 10247 scaffolds with 4% of residual gaps. Masked regions are represented with lowercase characters (soft-masking); gaps in the assembly are represented with Ns.</p>

<p>The genome was annotated using both EVM/PASA pipeline and cufflinks/transdecoder. Both these annotation have their strength. Cufflinks provides a better representation of transcript diversity, limits and TSSs positions. Conversely, EVM generates more robust predicted proteins, especially in regard to frameshifts, etc… A unified annotation with unified IDs (BL00000) was subsequently built with a suffix indicative of their origin (BL00000_evm0 or BL00000_cuff0). All three annotations are available as GTFs. </p>

<table>
  <thead>
    <tr>
      <th>File</th>
      <th>Dataset</th>
      <th>Format</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong><a href="https://www.dropbox.com/s/hoor1hcv90c2la7/Bl71nemr.fa.gz?dl=1">Bl71nemr.fa</a></strong></td>
      <td>Genome assembly</td>
      <td>Fasta</td>
    </tr>
    <tr>
      <td><strong><a href="https://www.dropbox.com/s/4o1orqba6c1xrmy/Bla_annot-FINAL_v4_names.gtf.gz?dl=1">Bla_annot-FINAL_v4_names.gtf</a></strong></td>
      <td>Merged annotation (EVM+PASA)</td>
      <td>GTF format</td>
    </tr>
    <tr>
      <td> </td>
      <td> EVM/PASA annotation </td>
      <td>GTF format </td>
    </tr>
    <tr>
      <td> </td>
      <td>Cufflinks annotation</td>
      <td>GTF format </td>
    </tr>
    <tr>
      <td> </td>
      <td>Reference transcripts</td>
      <td>Fasta</td>
    </tr>
    <tr>
      <td> </td>
      <td>Reference proteins</td>
      <td>Fasta</td>
    </tr>
  </tbody>
</table>

<h2 id="genome-browser">Genome browser</h2>

<p>The european amphioxus genome assembly and annotation are available as a <a href="https://genome.ucsc.edu/goldenPath/help/hgTrackHubHelp.html">UCSC track hub</a>. The RNA-seq, ATAC-seq and Chip-seq available in the browser is not part of the bulk of genome data provided to the consortium and is a courtesy of JL Skarmeta’s lab. To enable access, you can either: </p>

<ul>
  <li>
    <p>Use the following link to a UCSC session which also includes functional genomics data. </p>
  </li>
  <li>
    <p>set up the hub by adding the hub address <code class="highlighter-rouge">http://zoo-animalia.zoo.ox.ac.uk/braLan/hub.txt</code> to <code class="highlighter-rouge">My Data &gt; Track Hubs &gt; My hubs</code> in UCSC menu. </p>
  </li>
</ul>

<h2 id="contacts">Contacts</h2>

<p>Please do not hesitate to contact us if you have a specific question: </p>

<p><a href="http://www.obs-banyuls.fr/annuaire/fiche.php?id=184">Hector Escriva</a>, Observatoire Océanologique de Banyuls-sur-Mer, France   </p>

<p><a href="http://www.zoo.ox.ac.uk/people/view/marletaz_f.htm">Ferdinand Marlétaz</a>, Department of Zoology, University of Oxford, UK</p>

<p><a href="https://www.upo.es/CABD/GomezSkarmetaLab/people.html">Nacho Maeso &amp; José-Luis Skarmeta</a>, CABD, Universidad Pablo de Olavide, Sevilla, Spain</p>

<p><a href="http://www.crg.eu/en/manuel_irimia">Manu Irimia</a>, CRG, Barcelona, Spain</p>


</article>


  

      <div class="footer">
        <p>
          &copy; 2015. All rights reserved.
        </p>
      </div>
    </div>

  </body>
</html>